app/controllers/admin/organisation_translations_controller.rb

```diff
@@ -20,6 +20,7 @@ def get_layout
   end
 
   def create_redirect_path
+    binding.pry
```

Comment:
can we clean this up from the commit history please

Reply:
sure sorry missed it ..